crates/supervisor/core/src/lib.rs

```diff
@@ -6,3 +6,25 @@ pub use supervisor::{Supervisor, SupervisorError, SupervisorService};
 
 mod rpc;
 pub use rpc::SupervisorRpc;
+
+use async_trait::async_trait;
+
+/// The [NodeActor] is an actor-like service for the supervisor.
```

Comment:
```suggestion
/// The [`NodeActor`] is an actor-like service for the supervisor.
```
let's code format code, same like `std` library docs do